src/react_native/mmkv.cljs

```diff
@@ -0,0 +1,103 @@
+(ns react-native.mmkv
+  (:require
+    ["react-native-mmkv" :refer [MMKV]]
+    [cognitect.transit :as transit]
+    [native-module.core :as native-module]
+    [oops.core :as oops]
+    [taoensso.timbre :as log])
+  (:refer-clojure :exclude [set]))
+
+;; Create a single MMKV instance to be used throughout the app with the path from native code
+(defonce ^:private storage
+  (let [mmkv-path (try
+                    (native-module/get-mmkv-storage-path)
+                    (catch :default _
```

Comment:
@qfrank what if this call throws in a release build, will it work to use the fallback path `/tmp/test-mmkv`? Seems better to only fallback to the `/tmp` path if it's not a release build. Wdyt?